### PR TITLE
fix(halo-theme): fix import native-element path

### PR DESCRIPTION
**Pull request by dtanp-rft**
## Description
`import '@refinitiv-ui/halo-theme/dark/imports/native-elements';` does not work anymore because we don't export this path in package.json.

Error message is
```
Error: Module not found: Error: Package path ./dark/import/native-elements is not exported from package C:\Users\UC255417\w\angular13\node_modules\@refinitiv-ui\halo-theme (see exports field in C:\Users\UC255417\w\angular13\node_modules\@refinitiv-ui\halo-theme\package.json)
```

We should ensure that existing app who use import '@refinitiv-ui/halo-theme/dark/imports/native-elements'; won't break after upgrading to EF5.

## Type of change
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by dtanp-rft** [comment]
@goremikins I have screw up the old PR so I create this one as a replacement :')

After review this again today, if we add `exports`, we will need to add many export paths e.g. `/src/`, `/imports/`, `/light/`, `/dark/`. All of this is for one benefit which is developer doesn't need to have `/imports/` when the import `native-elements`.

I think this is going to blot the package.json and don't feel that it's worth it. So, don't use `exports` field may be a better solution.

Can you review and approve this PR if agree?

cc @Theeraphat-Sorasetsakul

**Comment by dtanp-rft** [comment]
I'll update halo native-elements importing in the new elf-docs after this PR approved and we have released this.

**Comment by dtanp-rft** [comment]
Confirmed with Aleks offline and this PR is OK. Will merge it today.

**Review by Sakchai-Refinitiv — APPROVED** [review]

**Pull request merged**
